### PR TITLE
#Homework3

### PR DESCRIPTION
**Pull request by Petrus9107**

### CODE REVIEW

File: module3/homework/passwordCheck/validation.hpp [hunk 1]
```diff
@@ -1,2 +1,18 @@
-// TODO: I'm empty :) Put enum and function headers here.
-// Don't forget the header guard
\ No newline at end of file
+#pragma once
+#include <iostream>
+#include <string>
```
Comment (WBigaj) [inline module3/homework/passwordCheck/validation.hpp:3]: Newline between pragma and includes would be nice

File: module3/homework/vectorOfSharedPointers/vectorFunctions.cpp [hunk 2]
```diff
@@ -0,0 +1,42 @@
+#include "vectorFunctions.hpp"
+#include <iostream>
```
Comment (WBigaj) [inline module3/homework/vectorOfSharedPointers/vectorFunctions.cpp:2]: Header includes and standard library includes should be in another paragraph.

File: module3/homework/passwordCheck/validation.cpp [hunk 3]
```diff
@@ -1,2 +1,69 @@
 #include "validation.hpp"
-// TODO: Put implementations here
\ No newline at end of file
+
+#include <stdlib.h>
```
Comment (ziobron) [inline module3/homework/passwordCheck/validation.cpp:3]: modern version of this library is cstdlib. But I guess that it's not used here. Please remove this include.

**Comment by ziobron** [comment]
{ "requirement_ids": [1995, 1996, 1998] }
7 XP za Podstawy#3 – passwordCheck (1995)
4 XP za Podstawy#3 – passwordCheck bonusowe wymagania (1996)
5 XP za Podstawy#3 – vectorOfSharedPointers (1998)

**Pull request closed without merging**